csi/controllerserver.py

```diff
@@ -145,8 +145,13 @@ def CreateVolume(self, request, context):
 
                 # The external volume should be used as kadalu host vol
 
-                # TODO: handle the case where host-volume is full
-                # can-be-fixed-by-an-intern
+                if is_hosting_volume_free(ext_volume['name'], pvsize) is False:
```

Comment:
any reason for not using `not is_hosting_volume_free` construct rather than comparing a boolean?

Reply:
Nothing specific but it was my first intuition. Ill use above format from now on.